src/builderGame/components/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  Input,
+  Popover,
+  PopoverArrow,
+  PopoverBody,
+  PopoverCloseButton,
+  PopoverContent,
+  PopoverTrigger,
+} from '@chakra-ui/react';
+import { DayPicker } from 'react-day-picker';
+import { Control, Controller } from 'react-hook-form';
+import { InputContainer } from '../InputContainer';
+import { format } from 'date-fns';
+import { PlaceAndDate } from '../SelectPlaceAndDateForm/useSelectPlaceAndDateForm';
+
+type Props = {
+  control: Control<PlaceAndDate, any>;
+};
+
+export const DatePicker: React.FC<Props> = ({ control }) => {
+  return (
+    <InputContainer label="test">
+      <Controller
+        control={control}
+        name="date"
+        rules={{ required: true }}
+        render={({ field: { onChange, name, value } }) => (
+          <Popover>
+            <PopoverTrigger>
+              <Input
+                value={format(value, 'dd/MM/yyyy')}
+                name={name}
+                onChange={() => {}}
```

Comment:
Is it needed?

Reply:
It is a work in progress because the user will be able to change the date with the `<DatePicker />` or by writing the date into this input